MapboxCoreNavigation/RouteController.swift

```diff
@@ -407,9 +434,15 @@ extension RouteController {
 
     func willReroute(notification: NSNotification) {
         _ = enqueueRerouteEvent()
+        sendFasterRouteEvent()
```

Comment:
This sends a faster route event every time a user is about to reroute.

Shouldn't we send this from the faster reroute callback, only if a new route is presented to the user?

Reply:
In fact, this is called before `enqueueFoundFasterRouteEvent()`, which is called from `didReroute()`.